src/components/material/constants.ts

```diff
@@ -0,0 +1,28 @@
+export const INCLUDE_EXCLUDE_TIPPY = {
```

Comment:
Can we rename it to `INCLUDE_EXCLUDE_COMMIT_TIPPY`

Reply:
Done